src/Corporation/ui/IndustryProductEquation.tsx

```diff
@@ -13,12 +13,12 @@ export function IndustryProductEquation(props: IProps): React.ReactElement {
     if (reqAmt === undefined) continue;
     reqs.push(String.raw`${reqAmt}\text{ }${reqMat}`);
   }
-  const prod = props.division.prodMats.slice();
+  const prod = props.division.prodMats.map((p) => `1\\text{ }${p}`);
   if (props.division.makesProducts) {
-    prod.push(props.division.type);
+    prod.push("Products");
```

Comment:
```suggestion
    prod.push("1 Output Per Product");
```

hmm i cant think of a better way to say this but it should still be applicable that it takes all of X to make 1 Y right?

Reply:
I agree, this has more specific wording.
But it's not ideal still,
```5 Metal + 5 Energy + 2 Water + 4 Hardware => 1 RealEstate + 1 Output Per Product```
But each product consumes separate set of input ingredients, so it should probably be reworded like:
```5 Metal + 5 Energy + 2 Water + 4 Hardware => 1 RealEstate or Product```
or even
```(5 Metal + 5 Energy + 2 Water + 4 Hardware) * (1 + Products) => 1 RealEstate + 1 Output Per Product```

Reply:
another option we could consider is two lines 
```
5 Metal + 5 Energy + 2 Water + 4 Hardware => 1 RealEstate
5 Metal + 5 Energy + 2 Water + 4 Hardware => 1 Product
```